rust/kcl-lib/src/std/array.rs

```diff
@@ -223,15 +236,20 @@ async fn call_reduce_closure(
 ///
 /// ```no_run
 /// arr = [1, 2, 3]
-/// new_arr = push(arr, 4)
+/// new_arr = push(arr, item = 4)
 /// assert(new_arr[3], isEqualTo = 4, tolerance = 0.1, error = "4 was added to the end of the array")
 /// ```
 #[stdlib {
     name = "push",
+    keywords = true,
+    unlabeled_first = true,
+    args = {
+        array = { docs = "The array which you're adding a new item to." },
+        item = { docs = "The new item to add to the array" },
```

Comment:
Include in snippet?

Reply:
Mandatory args will always be in the snippet. include_in_snippet is only useful for optional args, which default to not included in snippet.